src/main/java/com/librato/metrics/client/Sanitizer.java

```diff
@@ -3,9 +3,20 @@
 import java.util.regex.Pattern;
 
 /**
- * Filters out unwanted characters
+ * Filters out unwanted characters and truncates
  */
-public interface Sanitizer {
+public abstract class Sanitizer {
+
+    private static String sanitize(final String unclean, final Pattern disallowedChars, final int maxLength) {
```

Comment:
![image](https://user-images.githubusercontent.com/6645714/34259809-83e6e568-e618-11e7-8c52-cdb7d0fd9f78.png)

Reply:
I was going for an evangelical/cult point of view, but I like where you took it